.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish nightly
+on:
+  schedule:
+    - cron: '0 7 * * *'
+  workflow_dispatch: {}
+  pull_request: # TMP TODO DELETE ME
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - run: yarn
+
+      - name: bash $GITHUB_WORKSPACE/.github/scripts/bump-nightly-version.sh
+      - uses: JS-DevTools/npm-publish@v1
```

Comment:
I believe we want to peg to a specific hash since the tag can be changed at any time

Reply:
aside: do we actually need this? it seems like we don't use any of their touted improvements over just `npm publish` and if we can avoid giving out our NPM_TOKEN with a line or two, that feels worth it :)

Reply:
> Only publishes if the version number in package.json differs from the latest on NPM

hmm..does npm even let you republish the latest version?

Reply:
not that I'm aware of. but also aren't we publishing a brand new version every time guaranteed?